src/main/java/org/fundacionjala/coding/ketty/SpinWords.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.ketty;
+
+/**
+ * this is class verify if a string is a SPIN WORD.
+ */
+
+public class SpinWords {
+
+    private static final int LIMIT_LETTER = 5;
+
+    /**
+     * @param phrase it method for verify if a string can to be a Spin Word.
+     * @return a string.
+     */
+
+    public String getSpinWord(String phrase) {
+        StringBuilder newPhrase = new StringBuilder();
+        String[] word = phrase.split(" ");
+
+        for (String wordPart : word) {
+            if (wordPart.length() < LIMIT_LETTER) {
+                newPhrase.append(" " + wordPart);
+
+            } else {
+
+                StringBuilder reverse = new StringBuilder(wordPart);
+                newPhrase.append(" " + reverse.reverse().toString());
+            }
+        }
+        return newPhrase.toString().trim();
```

Comment:
This may be refactored like this.
```java
    String getSpinWord(final String phrase) {
        StringJoiner newPhrase = new StringJoiner(DELIMITER_WHITE_SPACE);
        String[] word = phrase.split(DELIMITER_WHITE_SPACE);
        for (String wordPart : word) {
            newPhrase.add(wordPart.length() < LIMIT_LETTER ? wordPart : new StringBuilder(wordPart).reverse());
        }
        return newPhrase.toString();
    }
```